test/arrays.js

```diff
@@ -583,4 +583,19 @@
     assert.deepEqual(_.chunk([10, 20, 30, 40, 50, 60, 70], 2), [[10, 20], [30, 40], [50, 60], [70]], 'chunk into parts of less then current array length elements');
     assert.deepEqual(_.chunk([10, 20, 30, 40, 50, 60, 70], 3), [[10, 20, 30], [40, 50, 60], [70]], 'chunk into parts of less then current array length elements');
   });
+
+  QUnit.test('remove', function(assert) {
+    var result = [1, 2, 3, 4]
+    _.remove(result, function(obj) {return(obj == 2)})
+    assert.deepEqual(result, [1, 3, 4]);
```

Comment:
This is good. I now understand that your tests in the previous PR where intended differently than I thought.

There is just one thing missing: these tests aren't verifying yet that `result` is also returned from `_.remove`. You can fix this as follows:

```suggestion
    var result = [1, 2, 3, 4];
    assert.strictEqual(_.remove(result, function(obj) {return(obj == 2)}), result);
    assert.deepEqual(result, [1, 3, 4]);
```